digital-credentials/index.html

```diff
@@ -0,0 +1,5 @@
+<!doctype html>
+<script>
+  const { hash } = window.location;
+  window.location = `https://w3c.github.io/digital-credentials/${hash}`;
```

Comment:
```suggestion
  window.location = `https://w3c-fedid.github.io/digital-credentials/${hash}`;
```

Reply:
@marcoscaceres Can you respond here? What should be the right destination for this?